99_post_install.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+
+KUBECONFIG=${KUBECONFIG:-ocp/auth/kubeconfig}
+IFCFG_INTERFACE="assets/post-install/ifcfg-interface.template"
+IFCFG_BRIDGE="assets/post-install/ifcfg-bridge.template"
+
+export bridge="${bridge:-brext}"
+
+create_bridge(){
+  echo "Deploying Bridge ${bridge}..."
+
+  FIRST_MASTER=$(oc get node -o custom-columns=IP:.status.addresses[0].address --no-headers | head -1)
+  export interface=$(ssh -q core@$FIRST_MASTER "ip r | grep default | grep -Po  '(?<=dev )(\S+)'")
```

Comment:
```suggestion
  export interface=$(ssh -q -o StrictHostKeyChecking=no core@$FIRST_MASTER "ip r | grep default | grep -Po  '(?<=dev )(\S+)'")
```
To prevent wait for the input like:
```
interface=$( ssh -q core@192.168.123.142 "ip r | grep default | grep -Po '(?<=dev)(\S+)'" )
The authenticity of host '192.168.123.142 (192.168.123.142)' can't be established.
ECDSA key fingerprint is SHA256:8nb/2Y1NfRhEgpWDIKmDnZLhxz/w+8gnebxD6qDpOeU.
Are you sure you want to continue connecting (yes/no)? ^C
```

Reply:
Fixed, thanks!